packages/protocol/test/governance/epochrewards.ts

```diff
@@ -0,0 +1,576 @@
+      })
+    })
+  })
+
+  describe('#getRewardsMultiplier()', () => {
+    const timeDelta = YEAR.times(10)
+    const expectedTargetTotalSupply = getExpectedTargetTotalSupply(timeDelta)
+    const expectedTargetRemainingSupply = SUPPLY_CAP.minus(expectedTargetTotalSupply)
+    let targetEpochReward: BigNumber
+    beforeEach(async () => {
+      await timeTravel(timeDelta.toNumber(), web3)
+      targetEpochReward = await epochRewards.getTargetEpochRewards()
+      targetEpochReward = targetEpochReward.plus(
+        await epochRewards.getTargetTotalEpochPaymentsInGold()
+      )
+    })
+
+    describe('when the target supply is equal to the actual supply after rewards', () => {
+      beforeEach(async () => {
+        await mockGoldToken.setTotalSupply(expectedTargetTotalSupply.minus(targetEpochReward))
+      })
+
+      it('should return one', async () => {
+        assertEqualBN(await epochRewards.getRewardsMultiplier(), toFixed(1))
+      })
+    })
+
+    describe('when the actual remaining supply is 10% more than the target remaining supply after rewards', () => {
+      beforeEach(async () => {
+        const actualRemainingSupply = expectedTargetRemainingSupply.times(1.1)
+        const totalSupply = SUPPLY_CAP.minus(actualRemainingSupply)
+          .minus(targetEpochReward)
+          .integerValue(BigNumber.ROUND_FLOOR)
+        await mockGoldToken.setTotalSupply(totalSupply)
+      })
+
+      it('should return one plus 10% times the underspend adjustment', async () => {
+        const actual = fromFixed(await epochRewards.getRewardsMultiplier())
+        const expected = new BigNumber(1).plus(
+          fromFixed(rewardsMultiplier.adjustments.underspend).times(0.1)
+        )
+        // Assert equal to 7 decimal places due to fixidity imprecision.
+        assert.equal(expected.dp(7).toFixed(), actual.dp(7).toFixed())
+      })
+    })
+
+    describe('when the actual remaining supply is 10% less than the target remaining supply after rewards', () => {
+      beforeEach(async () => {
+        const actualRemainingSupply = expectedTargetRemainingSupply.times(0.9)
+        const totalSupply = SUPPLY_CAP.minus(actualRemainingSupply)
+          .minus(targetEpochReward)
+          .integerValue(BigNumber.ROUND_FLOOR)
+        await mockGoldToken.setTotalSupply(totalSupply)
+      })
+
+      it('should return one minus 10% times the underspend adjustment', async () => {
+        const actual = fromFixed(await epochRewards.getRewardsMultiplier())
+        const expected = new BigNumber(1).minus(
+          fromFixed(rewardsMultiplier.adjustments.overspend).times(0.1)
+        )
+        // Assert equal to 7 decimal places due to fixidity imprecision.
+        assert.equal(expected.dp(7).toFixed(), actual.dp(7).toFixed())
+      })
+    })
+  })
+
+  describe('#updateTargetVotingYield()', () => {
+    const randomAddress = web3.utils.randomHex(20)
```

Comment:
Similar here as near the beginning of this file, might be worth renaming this to `mockReserveAddress` or similar for context.